code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1630,3 +1630,14 @@
 	description = "blue sparkles that get everywhere"
 	color = "#4040FF" //A blueish color
 	glitter_type = /obj/effect/decal/cleanable/glitter/blue
+
+/datum/reagent/pukonium
```

Comment:
Just because it's obtainable through bad cooking wouldn't mean that you can't obtain it from other means as well..it makes no sense to have another chemical that serves the same purpose. Just upgrade bad food.